thorlcr/activities/indexwrite/thindexwrite.cpp

```diff
@@ -354,6 +407,11 @@ class IndexWriteActivityMaster : public CMasterActivity
         CMasterActivity::getActivityStats(stats);
         stats.addStatistic(StNumDuplicateKeys, cummulativeDuplicateKeyCount);
     }
+
+    void deserializeStats(unsigned node, MemoryBuffer &mb) override
```

Comment:
minor:
missing virtual keyword.
But no point in defining this override, as same as base.

Reply:
I will delete - that was left there from trying to debug why the stats where not appearing properly.